test/CMakeLists.txt

```diff
@@ -53,4 +53,6 @@ target_link_libraries(${This} PUBLIC
     gmock
     pthread
     libCalculator
+    ${PISTACHE_LIBRARY}
```

Comment:
Here, you should be able to just write `pistache` or `pistache_static` instead of defining your own ${PISTACHE_LIBRARY}